### Instructions
#### It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

#### 1.Set up the hypothesis test.


In [4]:
import scipy.stats
import math
import pandas


 (H<sub>0</sub>): X̄ = 120 mm Hg 


 (H<sub>1</sub>): X̄ ≠ 120 mm Hg 

In [11]:
#### 2.Write down all the steps followed for setting up the test.

In [ ]:
#### A one sample, two tailed test is applied as H1 proposes that the difference in blood pressure goes either way. As this is a medical scenario, an alpha(α) of 0.01 is chosen, which translates to a significance level of 99%. As the population standard deviation, represented by σ, is known -- we can use the a one sample two tailed t test.

#### The formula for the test statistic is: T = (X̄ – μ) / σ/√n

In [12]:
population_mean= 120
sample_mean= 130.1
sample_size= 100
sample_std_dev= 21.21
α = 0.01/2 #two tailed test
α= 0.005



#### Calculate the test statistic by hand and also code it in Python. It should be 4.76190. We will take a look at how to make decisions based on this calculated value.

In [8]:
t_stat= (sample_mean-population_mean)/(sample_std_dev/math.sqrt(sample_size)) #calculating the test statistic
print(t_stat)

4.761904761904759


In [ ]:
#calculating the critical value with the degrees of freedom(df)

df= (n-1)
df= 100-1
df= 99



#### According to this <a href="<a href="https://openai.com">OpenAI</a>">T statistic reference table</a>, the critical values is: 
2.6269



#### Interpreting the results

#### As the test statistic is greater than the absolute critical value ( 4.762 > 2.627 ), we can therefore reject the null hypothesis. Therefore, there sufficient evidence to conclude that the mean systolic blood pressure of the sample is siginificantly different from that of the population.

#### We can also compare the chosen alpha with the p-value, which represents the possibility of obtaining the observed test statistic under the null hypothesis.

#### P-value = 0.00999

#### As the p-value is less than the chosen alpha (0.01>0.00999), we can reject the null hypothesis.


